.buildkite/pipeline.yml

```diff
@@ -98,6 +98,21 @@ steps:
       - step: "tests"
         allow_failure: false
 
+  - label: ":argo: Update image tag for Fleet Server"
+    trigger: serverless-gitops-update-stack-image-tag
+    async: true
+    branches: main
+    build:
+      env:
+        IMAGE_TAG: "$DOCKER_IMAGE_TAG"
+        SERVICE: fleet-controller
+        NAMESPACE: fleet-ci
+        IMAGE_NAME: fleet-server
+        COMMIT_MESSAGE: "gitops: update fleet-server tag to elastic/fleet-server@$GIT_ABBREV_COMMIT"
```

Comment:
I think `GIT_ABBREV_COMMIT` env. variable does not exist in the Buildkite context.

Probably, it should be used:
- GITHUB_PR_HEAD_SHA
- BUILDKITE_COMMIT

They contain the full sha of the commit. Example:
```
GITHUB_PR_HEAD_SHA="74556f61420168ef34592455a2bd42e4c4e01dc9"
BUILDKITE_COMMIT="74556f61420168ef34592455a2bd42e4c4e01dc9"
```

Reply:
If those values needs to be calculated, it looks like it should be created this step dynamically (following the example in Kibana repo).

Here there is another example of creating steps dynamically:
- https://github.com/elastic/elastic-package/blob/main/.buildkite/pipeline.yml#L38-L39
- https://github.com/elastic/elastic-package/blob/main/.buildkite/pipeline.trigger.integration.tests.sh